example/lib/widgets/initialize.dart

```diff
@@ -28,17 +29,18 @@ class _InitializeState extends BaseSampleWidgetState<Initialize> {
     String androidTvNoticeIdText = _androidTvNoticeIdController.text;
     String? androidTvNoticeId = androidTvNoticeIdText.isEmpty ? null : androidTvNoticeIdText;
 
-    await DidomiSdk.initialize(
-        _apiKeyController.text,
+    DidomiInitializeParameters didomiInitializeParameters = DidomiInitializeParameters(
+        apiKey: _apiKeyController.text,
         disableDidomiRemoteConfig: _disableRemoteConfigValue,
         languageCode: languageCode,
         noticeId: noticeId,
-        androidTvNoticeId: androidTvNoticeId,
+        tvNoticeId: androidTvNoticeId,
```

Comment:
for consistency with `androidTvEnabled`, should `tvNoticeId` have the `android` prefix as well?

Reply:
I agree, we should keep `android` prefix as it is only used for Android. This parameter is ignored on iOS.

Reply:
This is because on iOS, mobile and CTV are 2 different targets, so it is easy to handle the 2 use cases ; while on Android, the same apk can be run on both mobile and CTV.

Reply:
we are using `tvNoticeId` from the native code: https://gitlab.com/didomi/sdks/mobile/android/-/blob/master/android/src/main/java/io/didomi/sdk/DidomiInitializeParameters.kt#L72